### PR TITLE
Added an FAQ page with initial content

### PR DESCRIPTION
**Pull request by csharpfritz**
Addresses #81 - adds a static FAQ page

### CODE REVIEW

File: src/live.asp.net/Views/Home/Faq.cshtml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+﻿@*
+    For more information on enabling MVC for empty projects, visit http://go.microsoft.com/fwlink/?LinkID=397860
```
Comment (Eilon) [inline src/live.asp.net/Views/Home/Faq.cshtml]: Pretty sure you can delete this comment :smile:

File: src/live.asp.net/Views/Home/Faq.cshtml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  <dd>
+    The links will always be made available in the next day or two after the standup on the <a href="http://blogs.msdn.com/webdev">MSDN WebDev blog</a>.  All notes are tagged
+    under <a href="https://blogs.msdn.microsoft.com/webdev/tag/communitystandup/">CommunityStandup</a>.
+  </dd>
+  <dt>When will my favorite feature be available in ASP.NET Core?</dt>
+  <dd>
+    The roadmap of currently planned features is available on the <a href="https://github.com/aspnet/home/wiki/roadmap">ASP.NET GitHub Wiki</a>.
+  </dd>
+  <dt>Can you help me with this code?</dt>
+  <dd>Post these type of support questions to <a href="http://stackoverflow.com/questions/tagged/asp.net-core">StackOverflow with a tag of "asp.net-core"</a> and members of the team can locate it and answer it for all to see and learn from.</dd>
```
Comment (Eilon) [inline src/live.asp.net/Views/Home/Faq.cshtml]: I'd remove the word `support` from this and just say `these types of questions`. SO is not for "support".

File: src/live.asp.net/Views/Home/Faq.cshtml (hunk 1 above)
Comment (csharpfritz) [inline src/live.asp.net/Views/Home/Faq.cshtml]: Removed... thanks!

File: src/live.asp.net/Views/Home/Faq.cshtml (hunk 2 above)
Comment (csharpfritz) [inline src/live.asp.net/Views/Home/Faq.cshtml]: rephrased as requested

**Comment by Eilon** [comment]
:shipit: from me, but not sure if that counts for this repo :smile:

**Pull request closed without merging**